UI/BuildDesignatorWnd.cpp

```diff
@@ -777,6 +805,18 @@ void BuildDesignatorWnd::BuildSelector::PopulateList() {
     auto default_font = ClientUI::GetFont();
     const GG::Pt row_size = m_buildable_items->ListRowSize();
 
+    // populate list with fixed projects
+    {
+        auto stockpile_row = GG::Wnd::Create<ProductionItemRow>(row_size.x, row_size.y,
+            ProductionQueue::ProductionItem(BT_STOCKPILE), m_empire_id, m_production_location);
```

Comment:
indent or move all params to start on new line

Reply:
ok